doc/api/child_process.markdown

```diff
@@ -176,8 +176,8 @@ replace the existing process and uses a shell to execute the command.*
 
 ### child_process.execFile(file[, args][, options][, callback])
 
-* `file` {String} The filename of the program to run
-* `args` {Array} List of string arguments
+* `file` {String} An absolute path to an executable file.
+* `args` {Array} List of string arguments. **Note:** Path arguments must also be absolute.
```

Comment:
Interpretation of arg vector is ENTIRELY up to the command being execed, and any args, which may or not be paths, will be looked up as the command wishes. This will NOT usually mean that file names need to be absolute.

```
> fs.writeFileSync("hi.txt", "hi")
undefined
> child_process.execFile('cat', ["hi.txt"], function(_, stdout){console.log(stdout)});null;
null
> hi
```

Its worth mentioning that since the shell is not called, no shell behaviours such as globbing or i/o redirection will occur.